pkg/multiTenancyPlugins/naming/nameScoping.go

```diff
@@ -129,3 +131,150 @@ func (nameScoping *DefaultNameScopingImpl) Handle(command utils.CommandEnum, clu
 	}
 	return nil
 }
+
+func CheckContainerReferences(cluster cluster.Cluster, tenantId string, containerConfig *dockerclient.ContainerConfig) error {
+	log.Debugf("CheckContainerReferences containerConfig: %+v", containerConfig)
+	// create arrays of container references to pass to getIDsFromContainerReferences
+	// create volumesFrom array
+	volumesFrom := containerConfig.HostConfig.VolumesFrom
+
+	// create affinity array
+	affinityRef := make([]string, 0)
+	env := containerConfig.Env
+	var affinityIndex int
+	affinityLabelFirst := false
+	for i, e := range env {
```

Comment:
@ezrasilvera : have not seen any specifically stating that you can not have more than one affinity.  But it logically does not make any sense to me.  Affinity is meant to co-locate contains on the same host.  If more than one affinity is supplied but all the containers are on the same host the affect identical.  How would you handle affinity to multiple containers not located on the same host? Lets discuss.